### PR TITLE
Change reduce function docs for readability

### PR DESCRIPTION
**Pull request by anshbansal**
A huge paragraph was not readable. Seems more readable.

I am not sure why the http is being changed to rhttp.

### CODE REVIEW

**Comment by puredanger** [comment]
This project does not accept pull requests. See the contributing page for more details - http://clojure.org/contributing.

However, in this case, we would decline this change as the current concise description is preferred.

**Comment by anshbansal** [comment]
@puredanger 
Thanks for the reply.

Can at least two new lines be added before each of the val conditions. It was a handful to read all the ifs and nested ifs. Wasn't clear at the beginning which were the two top ifs.

**Comment by anshbansal** [comment]
Just asking the above for future reference so I can contribute to docs which is acceptable.

File: src/clj/clojure/core.clj [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ;   Copyright (c) Rich Hickey. All rights reserved.
 ;   The use and distribution terms for this software are covered by the
-;   Eclipse Public License 1.0 (http://opensource.org/licenses/eclipse-1.0.php)
+;   Eclipse Public License 1.0 (rhttp://opensource.org/licenses/eclipse-1.0.php)
```
Comment (jjconti) [inline src/clj/clojure/core.clj]: Careful.

**Comment by anshbansal** [comment]
Closing as this repository of Clojure does not accept pull requests.

@jjconti Thanks for pointing that out. Will take care.

**Pull request closed without merging**